web-components/src/components/form/form.mock.ts

```diff
@@ -12,4 +14,6 @@ export const bottomTextMapping = {
   countryLabel: 'Country',
   stateLabel: 'State / Region',
   postalCodeLabel: 'Postal Code',
+  locationStatePlaceholderLabel: 'State / Region',
```

Comment:
what's the difference between this and `stateLabel`?

Reply:
The mock is incorrect; it should be `Please choose a state,` I'll update it, thanks.